README.md

```diff
@@ -5,88 +5,109 @@
+The Klaviyo Android SDK allows developers to incorporate Klaviyo analytics and push notification functionality
+in their native Android applications. The SDK assists in identifying users and tracking user events via the 
+latest Klaviyo RESTful client APIs. To reduce performance overhead, API requests are queued and sent in batches. 
+The queue is persisted to local storage so that data is not lost if the device is offline or the app is terminated.
+
 Once integrated, your marketing team will be able to better understand your app users' needs and
 send them timely push notifications via FCM.
 
 > ⚠️ **We support Android API level 23 and above** ⚠️
 
 ## Installation
-
-1. Include the [JitPack](https://jitpack.io/#klaviyo/klaviyo-android-sdk) repository in your
-   project's build file
-   ```kotlin
-   // build.gradle.kts
-   allprojects {
-       repositories {
-           maven(url = "https://jitpack.io")
+1. Include the [JitPack](https://jitpack.io/#klaviyo/klaviyo-android-sdk) repository in your project's build file
+   <details>
+      <summary>Kotlin DSL</summary>
+
+      ```kotlin
+      // build.gradle.kts
+      allprojects {
+          repositories {
+              maven(url = "https://jitpack.io")
+          }
+      }
+      ```
+   </details>
+
+   <details open>
+      <summary>Groovy</summary>
+
+      ```groovy
+      // build.gradle
+      allprojects {
+          repositories {
+              maven { url "https://jitpack.io" }
+          }
+      }
+      ```
+   </details>
+
+2. Add the dependencies to your app's build file
+   <details>
+      <summary>Kotlin DSL</summary>
+
+      ```kotlin
+      // build.gradle.kts
+      dependencies {
+          implementation("com.github.klaviyo.klaviyo-android-sdk:analytics:2.0.0")
+          implementation("com.github.klaviyo.klaviyo-android-sdk:push-fcm:2.0.0")
+      }
+      ```
+   </details>
+
+   <details open>
+      <summary>Groovy</summary>
+
+      ```groovy
+       // build.gradle
+       dependencies {
+           implementation "com.github.klaviyo.klaviyo-android-sdk:analytics:2.0.0"
+           implementation "com.github.klaviyo.klaviyo-android-sdk:push-fcm:2.0.0"
        }
-   }
-   ```
-   ```groovy
-   // build.gradle
-   allprojects {
-       repositories {
-           maven { url "https://jitpack.io" }
-       }
-   }
-   ```
-2. Add these dependencies to your app's build file
-   ```kotlin
-   // build.gradle.kts
-   dependencies {
-       implementation("com.github.klaviyo.klaviyo-android-sdk:analytics:2.0.0")
-       implementation("com.github.klaviyo.klaviyo-android-sdk:push-fcm:2.0.0")
-   }
-   ```
-   ```groovy
-   // build.gradle
-   dependencies {
-       implementation "com.github.klaviyo.klaviyo-android-sdk:analytics:2.0.0"
-       implementation "com.github.klaviyo.klaviyo-android-sdk:push-fcm:2.0.0"
-   }
-   ```
-
-## Analytics SDK
-
-### Configuration
-
-The SDK must be configured with the public API key for your Klaviyo account.
```

Comment:
Is this line intended by after the collapsable codeblock is closed? Looks like in the diff its within the codeblock still

Reply:
Not 100% sure what you mean, but you can toggle on the "rich diff" or just view the markdown [rendered](https://github.com/klaviyo/klaviyo-android-sdk/blob/feat/readme/README.md) to verify. The collapsible blocks look right to me

![Screenshot 2024-01-24 at 09 56 04](https://github.com/klaviyo/klaviyo-android-sdk/assets/5167687/3ca87fec-1c5e-4af6-9cc8-ab06a6db6533)

![Screenshot 2024-01-24 at 09 53 14](https://github.com/klaviyo/klaviyo-android-sdk/assets/5167687/5c8a0673-a64c-4a53-aa0e-0010603bc28e)

Reply:
Ya disregard, I had just misread the diff